alf/environments/utils.py

```diff
@@ -155,10 +156,17 @@ def create_environment(env_name='CartPole-v0',
     if nonparallel:
         # Each time we can only create one unwrapped env at most
         py_env = env_load_fn(env_name)
+        py_env.seed(np.random.randint(0, np.iinfo(np.int32).max))
```

Comment:
So according to my understanding, if np is deterministic, then the seeds for py_env will also be deterministic. The entire training/playing will have deterministic results, right? In issue #222, you mentioned that there are some other possible reasons for non-deterministic unittest results. Should we address those factors?

Reply:
envs  (gym) will behave deterministically when np is deterministic.  This pr does not solve the issue #222 . It just set seed for environment, and make settings as deterministic as possible when seed is set .